data/discord.json

```diff
@@ -13,20 +13,20 @@
       "permissions": []
     },
     "senior admin": {
-      "inherits": "maintainer",
-      "permissions": ["addao"]
-    },
-    "maintainer": {
       "inherits": "staff",
-      "permissions": []
+      "permissions": ["addao"]
     },
     "staff": {
       "inherits": "retmin",
       "permissions": ["userverify"]
     },
     "retmin": {
+      "inherits": "maintainer",
+      "permissions": ["tempban", "ban", "unban", "kick", "ticket", "whitelist", "toggleooc", "note", "staffban"]
+    },
+    "maintainer": {
       "inherits": "mentor",
```

Comment:
Should inherit from "staff", otherwise this PR removes a lot of perms from Maintainer.

Reply:
This is intentional